Recompiler: MOVZ / MOVN write only the low 64 bits of rd, like the R5900 (PCSX2: GPR.r[rd].UD[0] = GPR.r[rs].UD[0]);
they were translated as a 128-bit copy that also overwrote the upper half. Same fix as the God of War II port's
ddcc548. Changes the generated code: run the recomp, sync and build stages.

diff --git a/PS2Recomp/ps2xRecomp/src/lib/special_translator.cpp b/PS2Recomp/ps2xRecomp/src/lib/special_translator.cpp
index b01a937..93aa0c8 100644
--- a/PS2Recomp/ps2xRecomp/src/lib/special_translator.cpp
+++ b/PS2Recomp/ps2xRecomp/src/lib/special_translator.cpp
@@ -121,9 +121,10 @@ namespace ps2recomp
         case SPECIAL_SLTU:
             return fmt::format("SET_GPR_U64(ctx, {}, ((uint64_t)GPR_U64(ctx, {}) < (uint64_t)GPR_U64(ctx, {})) ? 1 : 0);", inst.rd, inst.rs, inst.rt);
         case SPECIAL_MOVZ:
-            return fmt::format("if (GPR_U64(ctx, {}) == 0) SET_GPR_VEC(ctx, {}, GPR_VEC(ctx, {}));", inst.rt, inst.rd, inst.rs);
+            // gow-native-pc: the R5900 moves only the low 64 bits (PCSX2: GPR.r[rd].UD[0] = GPR.r[rs].UD[0]).
+            return fmt::format("if (GPR_U64(ctx, {}) == 0) SET_GPR_U64(ctx, {}, GPR_U64(ctx, {}));", inst.rt, inst.rd, inst.rs);
         case SPECIAL_MOVN:
-            return fmt::format("if (GPR_U64(ctx, {}) != 0) SET_GPR_VEC(ctx, {}, GPR_VEC(ctx, {}));", inst.rt, inst.rd, inst.rs);
+            return fmt::format("if (GPR_U64(ctx, {}) != 0) SET_GPR_U64(ctx, {}, GPR_U64(ctx, {}));", inst.rt, inst.rd, inst.rs);
         case SPECIAL_MFSA:
             return fmt::format("SET_GPR_U32(ctx, {}, ctx->sa);", inst.rd);
         case SPECIAL_MTSA:
